bigtable_automation/java/dataflow/csv2bt-template-metadata.json

```diff
@@ -39,6 +39,13 @@
             "paramType": "TEXT",
             "isOptional": true
         },
+        {
+            "name": "bigtableMaxNumWorkers",
+            "label": "Bigtable maximum number of workers",
```

Comment:
nit:  dataflowMaxNumWorkers (the variable isn't BT related right?)

but if its churn, feel free to ignore too

Reply:
done